web/src/javascript/drawMap.js

```diff
@@ -65,13 +65,25 @@ export function getPlayerColor(players, currentPlayer) {
 
 export function drawPlayersOnMap(players, canvas) {
   const context = canvas.getContext("2d");
+  let cellArray = [];
+  let cellInfo;
   for(const player of players) {
     const color = getPlayerColor(players, player);
     for(const cell of player.cells) {
-      drawCircle(context, cell.position, cell.radius, color);
-      writeCellTeamName(player.name, context, cell.position)
+      cellInfo = {
+        position: cell.position,
+        radius: cell.radius,
+        color: color,
+        playerName: player.name
+      };
+      cellArray.push(cellInfo);
     }
   }
+  cellArray.sort();
```

Comment:
You could only use the `immutable-sort` here (already `require`d in this file), something like:
```js
const cells = [].concat.apply([], players.map(player => player.cells);
const cellsToDraw = sort(cells, (a, b) => a.radius - b.radius); // draw bigger cells on top of smaller ones
```